### PR TITLE
Allow Android Versions

### PR DESCRIPTION
**Pull request by JohnTranQUT**
### Description

Allow android phones onto airship impostor server

### CODE REVIEW

**Comment by js6pak** [comment]
If forte decided to change the protocol version, are you sure it will work?

**Comment by JohnTranQUT** [comment]
> If forte decided to change the protocol version, are you sure it will work?

I had not done anything to the protocol however I tested a few games with andriod/bluestacks and it appears to be running smoothly with the https://github.com/Impostor/Impostor/pull/372 .

I will report back as soon there are any concerns.

File: src/Impostor.Server/Net/Manager/ClientManager.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ internal partial class ClientManager
         private static readonly HashSet<int> SupportedVersions = new HashSet<int>
         {
             GameVersion.GetVersion(2021, 3, 25), // 2021.3.31s
+            GameVersion.GetVersion(2021, 4, 2), // 2021.4.2a andriod
```
Comment (js6pak) [inline src/Impostor.Server/Net/Manager/ClientManager.cs:23]:
```suggestion
            GameVersion.GetVersion(2021, 4, 2), // 2021.4.2a
```

**Review by js6pak — CHANGES_REQUESTED** [review]

File: src/Impostor.Server/Net/Manager/ClientManager.cs (hunk 1 above)
Comment (JohnTranQUT) [reply src/Impostor.Server/Net/Manager/ClientManager.cs:23]: Yep, spelt it wrong. My bad.

**Review by js6pak — APPROVED** [review]

**Pull request merged**